webui/react/src/contexts/Store.tsx

```diff
@@ -48,7 +34,9 @@ export interface State {
   pinnedWorkspaces: Workspace[];
   pool: PoolOverview;
   resourcePools: ResourcePool[];
-  ui: UI;
+  ui: StateUI & {
+    omnibar: OmnibarState; // TODO: this should probably be out of ui
```

Comment:
Does that mean omnibar will be completely removed?

Reply:
by ui I mean not nested under ui. I'm not sure if it's the correct move or not yet

Reply:
Hmm if its flexible and easy to change in the future, either way is fine?